src/Inventory.zig

```diff
@@ -1947,6 +1965,79 @@ pub const Command = struct { // MARK: Command
 			return result;
 		}
 	};
+	const AddEnergy = struct { // MARK: AddEnergy
```

Comment:
Not needed anymore, please remove from this PR.

Likely health and hunger will behave differently in the future.

Reply:
through the sync operation that you added